.env.example

```diff
@@ -37,6 +37,11 @@ DB_SCHEMA=public
 # -- Mail settings --
 # -------------------
 
+# Whether to check email verification for new users.
+# 1: check
+# 0: Do not check
+CHECK_VERIFIED=0
```

Comment:
Please change this name to `NEW_ACCOUNT_EMAIL_VERIFICATION` and use `0` for disable and `1` for enable.